__tests__/__system__/cli/profiles/__snapshots__/cli.profile.create.cics-profile.integration.test.ts.snap

```diff
@@ -29,56 +32,53 @@ exports[`Create cics Profile Success scenarios should display create cics profil
       Specifies the name of the new cics profile. You can load this profile by using
       the name on commands that support the \\"--cics-profile\\" option.
 
- REQUIRED OPTIONS
- ----------------
+ OPTIONS
+ -------
 
-   --host  | -H (string)
+   --host | -H (string)
 
       The CMCI server host name
 
-   --user  | -u (string)
+   --port | -P (number)
 
-      Your username to connect to CICS
-
-   --password  | -p (string)
+      The CMCI server port
 
-      Your password to connect to CICS
+      Default value: 1490
```

Comment:
Do we want to remove the default value for `--port` in zowe-v2-lts?

Reply:
I haven't heard any complains about it, but might be a good thing to do going forward since ports are easily configurable

Reply:
Discussed with the team and we decided against this change since it seems that 1490 is the default port for CICS, not just a site-specific value.